packages/spore/src/cluster/index.ts

```diff
@@ -145,17 +145,22 @@ export async function transferSporeCluster(params: {
   id: ccc.HexLike;
   to: ccc.ScriptLike;
   tx?: ccc.TransactionLike;
+  scriptInfo?: SporeScriptInfoLike;
```

Comment:
Can we use the same args as `assertCluster` - `scripts` to allow auto choosing scriptInfo?